src/human.jl

```diff
@@ -43,6 +43,7 @@ precalc_human_dispersal(human::A, cellsize, take, human_exponent, dist_exponent)
     h, w = size(human)
     indices = broadcastable_indices(Int32, human)
     dist = (distances(human) .* cellsize) .^ dist_exponent
+    dist[1] = cellsize/6 * (sqrt(2) + log(1 + sqrt(2))) # mean distance from cell centre 
```

Comment:
what does this line do?

Reply:
we dont want distance from i to i to be zero because we get NaNs when we divide.

Reply:
Ok. You should make that 6 a named variable, magic numbers in the code (besides 0, 1 and 2) are confusing!

Reply:
Yeah use a function. (Weird that my email reply didn't show up inline here)